Resources/Prototypes/_Lavaland/Construction/explorer_suit.yml

```
@@ -0,0 +1,30 @@
+- type: constructionGraph
+  id: ExplorerSuit
+  start: start
+  graph:
+    - node: start
+      entity: LavalandEquipmentExplorerSuit
+      edges:
+      - to: reinforced1
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)